### ISSUE TITLE
C2022 error building hercules hercules-helper-windows project - Changed general.c and built and tested OK 

### ISSUE THREAD

**Issue by ejjeamore** [Hercules-Aethra/aethra#21]
wrljet suggested to post issue on aethra. Getting C2022 error in arethra directory in general1.c. 

In arethra directory I changed general1.c line 2145 to:
BYTE    tmp[CFC_MAX_OPSIZE+**30**];            /* (work field)              */

built all modules return = 0 and all tests passed.  I'm not sure if it will fix execution.
Don't use -forececlone to rebuild but I used -firewall

My interest is to just see if I can get it working. Interesting project.

**Comment by wrljet** [comment]
The fix here is most likely to just disable that warning, from yet another place.

That said I haven't looked at the code.  MSVC has cried wolf too often.

**Comment by ejjeamore** [comment]
Yeah, I hear ya. My fix was just to get it working.WilliamSent from my iPhoneOn Nov 9, 2024, at 11:39 AM, Bill Lewis ***@***.***> wrote:﻿
The fix here is most likely to just disable that warning, from yet another place.
That said I haven't looked at the code.  MSVC has cried wolf too often.

**Comment by wrljet** [comment]
I just tried this and "it works for me".  So I'm wondering, did you use the `-GitBranch develop` option with Hercules-Helper?

If not, you're getting ancient code, before the work we've already done to control these warnings.

Bill

**Comment by ejjeamore** [comment]
I took the "code" zip download off the website BillSent from my iPhoneOn Nov 9, 2024, at 6:14 PM, Bill Lewis ***@***.***> wrote:﻿
I just tried this and "it works for me".  So I'm wondering, did you use the -GitBranch develop option with Hercules-Helper?
If not, you're getting ancient code, before the work we've already done to control these warnings.
Bill

**Comment by ejjeamore** [comment]
How would I get the current code to download to try, I'm not real versed in repositories?
William
    On Sunday, November 10, 2024 at 10:19:25 AM CST, William Phillips ***@***.***> wrote:   

 I took the "code" zip download off the website 
Bill

Sent from my iPhone

On Nov 9, 2024, at 6:14 PM, Bill Lewis ***@***.***> wrote:



﻿


I just tried this and "it works for me". So I'm wondering, did you use the -GitBranch develop option with Hercules-Helper?

If not, you're getting ancient code, before the work we've already done to control these warnings.

Bill
